config/config_test.go

```diff
@@ -384,16 +384,16 @@ func TestParse(t *testing.T) {
 			false,
 		},
 		{
-			"exec_env_blacklist",
+			"exec_env_denylist",
 			`exec {
 				env {
-					blacklist = ["a", "b"]
+					Denylist = ["a", "b"]
```

Comment:
lowercase?

Reply:
good catch! The test still passes -- it's interesting that the keys are case insensitive 🤔